.github/workflows/release.yml

```diff
@@ -0,0 +1,194 @@
+  #       with:
+  #         name: ${{ runner.os }}-wheels
+  #         path: wheelhouse
+
+  # macos-release-wheel:
+  #   name: Build release wheels for macOS
+  #   runs-on: macos-latest
+  #   strategy:
+  #     matrix:
+  #       python-version: ['3.6', '3.7', '3.8']
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: actions/setup-python@v1
+  #       with:
+  #         python-version: ${{ matrix.python-version }}
+  #     - name: Build macOS wheels
+  #       if: github.event_name == 'release'
+  #       run: |
+  #         python3 --version
+  #         python3 -m pip install cython wheel setuptools
+  #         bazel build \
+  #           -c opt \
+  #           --copt -mmacosx-version-min=10.13 \
+  #           --linkopt -mmacosx-version-min=10.13 \
+  #           --noshow_progress \
+  #           --noshow_loading_progress \
+  #           --verbose_failures \
+  #           --test_output=errors \
+  #           build_pip_pkg
+  #         bazel-bin/build_pip_pkg artifacts
+  #         for f in artifacts/*.whl; do
+  #           delocate-wheel -w wheelhouse $f
+  #         done
+  #     - uses: actions/upload-artifact@v1
+  #       with:
+  #         name: ${{ runner.os }}-wheels
+  #         path: wheelhouse
+
+  # windows-release-wheel:
+  #   name: Build release wheels for Windows
+  #   runs-on: windows-latest
+  #   strategy:
+  #     matrix:
+  #       python-version: ['3.6', '3.7', '3.8']
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: actions/setup-python@v1
+  #       with:
+  #         python-version: ${{ matrix.python-version }}
+  #     - name: Build Windows wheels
+  #       shell: bash
+  #       if: github.event_name == 'release'
+  #       run: |
+  #         python -m pip install cython wheel setuptools
+  #         # TODO build here
+  #     - uses: actions/upload-artifact@v1
+  #       with:
+  #         name: ${{ runner.os }}-wheels
+  #         path: artifacts
+
+  # upload-wheels:
+  #   name: Publish wheels to PyPi
+  #   needs: [manylinux-release-wheel, macos-release-wheel, windows-release-wheel]
+  #   runs-on: ubuntu-18.04
+  #   steps:
+  #     - uses: actions/download-artifact@v1
+  #       with:
+  #         name: Linux-wheels
+  #         path: Linux-wheels
+  #     - uses: actions/download-artifact@v1
+  #       with:
+  #         name: macOS-wheels
+  #         path: macOS-wheels
+  #     - uses: actions/download-artifact@v1
+  #       with:
+  #         name: Windows-wheels
+  #         path: Windows-wheels
+  #     - run: |
+  #         set -e -x
+  #         mkdir -p dist
+  #         cp Linux-wheels/*.whl dist/
+  #         cp macOS-wheels/*.whl dist/
+  #         cp Windows-wheels/*.whl dist/
+  #         ls -la dist/
+  #         sha256sum dist/*.whl
+
+  #     - uses: pypa/gh-action-pypi-publish@master
+  #       if: github.event_name == 'release'
+  #       with:
+  #         user: __token__
+  #         password: ${{ secrets.pypi_token }}
```

Comment:
Please use another branch if you have a alternative approach.
And git allows us to do a diff between the to approaches.

Reply:
This is unlikely to be merged for many months, so keeping it as a separate branch would accumulate merge conflicts over time. Can I put it in the Github Wiki or a separate document instead maybe?

Reply:
It will be a branch with change only on `.github/workflows/release.yml`. There is few chance to have conflict. And if it there are, having the code commented in the same file will not help.

Reply:
Moved to: https://github.com/openzim/python-libzim/issues/25